torch/csrc/utils/throughput_benchmark.h

```diff
@@ -0,0 +1,181 @@
+#pragma once
+
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/jit/script/module.h>
+#include <pybind11/pybind11.h>
+
+#include <torch/csrc/jit/pybind_utils.h>
+
+#include <vector>
+#include <memory>
+
+namespace py = pybind11;
+
+namespace torch {
+namespace throughput_benchmark {
+
+/**
+ * The struct is used to provide results of a benchmark to the caller
+ * In the future all additional statics should be added here.
+ */
+struct BenchmarkExecutionStats {
+  float latency_avg_ms{-1};
+  int64_t num_iters{-1};
```

Comment:
Would be nice to add docs for this struct (what does `latency_avg_ms` mean in this case)